src/core/common/bit_vector.hpp

```diff
@@ -81,34 +81,23 @@ template <uint16_t N> class BitVector : public Equatable<BitVector<N>>
      * This method sets the mask of a given index.
      *
      * @param[in] aIndex  The index.
+     * @param[in] aSet    TRUE to set the mask, or FALSE to clear the mask.
      *
      */
-    void Set(uint16_t aIndex)
+    void Set(uint16_t aIndex, bool aSet)
```

Comment:
nit: how about `aValue`?
```suggestion
    void Set(uint16_t aIndex, bool aValue)
```